src/main/java/seedu/address/model/group/tasks/TaskInitializer.java

```diff
@@ -17,16 +27,18 @@ public static TaskList initializeTasks() throws TaskException {
         allTasks.add(new Deadline("Submit slides for op2", TaskStatus.NOT_DONE, TaskModule.CS2101, "29/10/2023 2359"));
         allTasks.add(new Deadline("Complete peer review", TaskStatus.NOT_DONE, TaskModule.CS2101, "02/11/2023 2359"));
         allTasks.add(new Todo("Research on the SCQA framework", TaskStatus.NOT_DONE, TaskModule.CS2101));
-        allTasks.add(new Deadline("plan for op2 ", TaskStatus.NOT_DONE, TaskModule.CS2101, "24/10/2023 2359"));
-        allTasks.add(new Deadline("submit UG", TaskStatus.NOT_DONE, TaskModule.CS2101, "09/11/2023 2359"));
+        allTasks.add(new Deadline("Plan for op2 ", TaskStatus.NOT_DONE, TaskModule.CS2101, "24/10/2023 2359"));
+        allTasks.add(new Deadline("Submit UG", TaskStatus.NOT_DONE, TaskModule.CS2101, "09/11/2023 2359"));
 
         // Create tasks for CS2103T
-        allTasks.add(new Todo("complete mid semester review form", TaskStatus.NOT_DONE, TaskModule.CS2103T));
-        allTasks.add(new Deadline("add demo screenshots to project notes", TaskStatus.NOT_DONE, TaskModule.CS2103T, "20/10/2023 2359"));
-        allTasks.add(new Deadline("release v1.3.trial jar file", TaskStatus.NOT_DONE, TaskModule.CS2103T, "27/10/2023 2359"));
-        allTasks.add(new Deadline("wrap up mileston 1.3", TaskStatus.NOT_DONE, TaskModule.CS2103T, "03/11/2023 2359"));
-        allTasks.add(new Deadline("finalise tp", TaskStatus.NOT_DONE, TaskModule.CS2103T, "17/11/2023 2359"));
-        allTasks.add(new Todo("update dg for each feature", TaskStatus.NOT_DONE, TaskModule.CS2103T));
+        allTasks.add(new Todo("Complete mid semester review form", TaskStatus.NOT_DONE, TaskModule.CS2103T));
+        allTasks.add(new Deadline("Add demo screenshots to project notes", TaskStatus.NOT_DONE, TaskModule.CS2103T,
+                "20/10/2023 2359"));
+        allTasks.add(new Deadline("Release v1.3.trial jar file", TaskStatus.NOT_DONE, TaskModule.CS2103T,
+                "27/10/2023 2359"));
+        allTasks.add(new Deadline("Wrap up mileston 1.3", TaskStatus.NOT_DONE, TaskModule.CS2103T, "03/11/2023 2359"));
+        allTasks.add(new Deadline("Finalise tp", TaskStatus.NOT_DONE, TaskModule.CS2103T, "17/11/2023 2359"));
+        allTasks.add(new Todo("Update dg for each feature", TaskStatus.NOT_DONE, TaskModule.CS2103T));
```

Comment:
Capitalise DG since UG was capitalised
```suggestion
        allTasks.add(new Todo("Update DG for each feature", TaskStatus.NOT_DONE, TaskModule.CS2103T));
```

Reply:
alright, fixed.